readme.md

```diff
@@ -81,7 +95,13 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
+```js
+Panda.prototype =  {
+  eat_bamboo: function() {
+    var self = this;
+    self.num_bamboo_eaten += 1;
+  }
+};
 
 ```
```

Comment:
If you write it the way you wrote it out, it will overwrite other things already stored in the Panda prototype. We tend to write out prototype methods like so...
```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```